docs/jwst/jump/arguments.rst

```diff
@@ -38,23 +38,22 @@ The ``jump`` step has five optional arguments that can be set by the user:
   not have its neighbors flagged. The goal is to prevent flagging jumps that would be too
   small to significantly affect the slope determination.  The default value is 10.
 
-*  ``--after_jump_flag_dn1``: A floating point value in units of DN that gives the
-   minimum ramp jump for after jump flagging for the 1st after ramp flagging.
+  After a jump of at least 'after_jump_flag_dn1' DN, frames up to 'after_jump_flag_time1'
```

Comment:
Should use "groups" instead of "frames". Frames has a very specific meaning for JWST data and correspond to individual detector readouts on-board, which is not always the same as what the software is dealing with on the ground (in cases of frame-averaging).